<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ThermometerDialog</class>
 <widget class="QDialog" name="ThermometerDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>578</width>
    <height>378</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Temperature Measurement</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="idGroupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>barcode</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="BarcodeWidget" name="barcodeWidget" native="true"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="devicesGroupBox">
     <property name="title">
      <string>devices</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="macLabel">
          <property name="text">
           <string>Address:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="deviceComboBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>1</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="scanButton">
          <property name="text">
           <string>Scan</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="connectButton">
          <property name="text">
           <string>Connect</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="disconnectButton">
          <property name="text">
           <string>Disconnect</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="measureGroupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>measure</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3" stretch="0,0">
      <item>
       <widget class="MeasureWidget" name="measureWidget" native="true"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QStatusBar" name="statusBar">
     <property name="sizeGripEnabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>BarcodeWidget</class>
   <extends>QWidget</extends>
   <header>widgets/BarcodeWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MeasureeWidget</class>
   <extends>QWidget</extends>
   <header>widgets/MeasureWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <includes>
  <include location="local">widgets/BarcodeWidget.h</include>
  <include location="local">widgets/MeasureWidget.h</include>
 </includes>
 <resources/>
 <connections/>
</ui>
